### PR TITLE
Visitor can see products menu on index page

### PR DESCRIPTION
**Pull request by Joberg95**
## Pt tracker
https://www.pivotaltracker.com/story/show/163901504

## Description 
- unit and feature tests are all green, refactored code to have more tests on the same code block
- list of products on index page

## Screenshot
Picture of index, WIP and no styling yet.
image

### CODE REVIEW

**Comment by Joberg95** [comment]
SOLVED! // There are some commits missing, i think its a branch issue.. will look into this with a coach later.

**Review by yanqiuzhang — APPROVED** [review]

File: features/visitor_can_see_products.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: Visitor can see products on landing page
+
+    As a visitor,
+    In order to get an overview of the restaurants products
+    I would like to be able to see a menu
+
```
Comment (amanda-mae) [inline features/visitor_can_see_products.feature:5]: is the formatting right under this? Should it all be indented one or two more than the Feature block?

**Review by aleciabenjamin — APPROVED** [review]
lgtm

File: features/visitor_can_see_products.feature (hunk 1 above)
Comment (Joberg95) [reply features/visitor_can_see_products.feature:5]: Fixed, see latest commit.

**Review by amanda-mae — APPROVED** [review]

**Review by boamatule — APPROVED** [review]

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+  def index
+  end
+end
```
Comment (faraznaeem) [inline app/controllers/welcome_controller.rb]: Remove this file

File: app/views/products/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.wrapper
+    %header
+        %h1 Menu
+
+        %ul
```
Comment (faraznaeem) [inline app/views/products/index.html.haml]: Formatting

File: features/step_definitions/basic_steps.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        FactoryBot.create(:product, product)
+    end 
+end
+  
+Given("I visit the landing page") do
+  visit root_path
+end
+
+Then("I should see {string}") do |price|
+  expect(page).to have_content price
```
Comment (faraznaeem) [inline features/step_definitions/basic_steps.rb]: change the variable name to something more generic

File: spec/factories/products.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :product do
+    name { "MyString" }
+    description {"MyText"}
+    price {1}
```
Comment (faraznaeem) [inline spec/factories/products.rb:5]: formatting

**Review by faraznaeem — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**